### PR TITLE
Changing some of our tests to use dotnet test

### PR DESCRIPTION
**Pull request by livarcocc**
We are missing the script tests.

### CODE REVIEW

File: src/Microsoft.DotNet.Files/project.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "keyFile": "../../tools/Key.snk"
+  },
+  "description": "Abstraction to interact with the file system and file paths.",
+  "dependencies": {
+    "NETStandard.Library": "1.0.0-rc2-23727",
```
Comment (davidfowl) [inline src/Microsoft.DotNet.Files/project.json]: Update these to match the new version.

File: NuGet.Config [hunk 2]
```diff
@@ -13,6 +13,7 @@
     <add key="corert" value="https://www.myget.org/F/dotnet/api/v3/index.json" />
     <add key="dotnet-buildtools" value="https://www.myget.org/F/dotnet-buildtools/api/v3/index.json" />
     <add key="fsharp-daily" value="https://www.myget.org/F/fsharp-daily/api/v3/index.json" />
+    <add key="coreclr-xunit" value="https://www.myget.org/F/coreclr-xunit/api/v2" />
```
Comment (davidfowl) [inline NuGet.Config]: omg more feeds

File: src/Microsoft.DotNet.Compiler.Common/project.json [hunk 3]
```diff
@@ -10,7 +10,8 @@
         "System.CommandLine": "0.1.0-e160119-1",
         "Microsoft.CodeAnalysis.CSharp":  "1.2.0-beta1-20160108-01",
         "Microsoft.DotNet.ProjectModel": "1.0.0-*",
-        "Microsoft.DotNet.Cli.Utils": "1.0.0-*"
+      "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
```
Comment (davidfowl) [inline src/Microsoft.DotNet.Compiler.Common/project.json]: nit: formatting

File: src/Microsoft.DotNet.Compiler.Common/project.json [hunk 4]
```diff
@@ -10,7 +10,8 @@
...
+        "Microsoft.DotNet.Files": {"version": "1.0.0-*", "type": "build"}
```
Comment (davidfowl) [inline src/Microsoft.DotNet.Compiler.Common/project.json]: This shouldn't be build time unless it's shared source.

**Comment by livarcocc** [comment]
@dotnet-bot Test CentOS7.1 Debug Build please
@dotnet-bot Test CentOS7.1 Release Build please
@dotnet-bot Test Ubuntu Debug Build please
@dotnet-bot Test Ubuntu Release Build please

**Pull request merged**